ui-admin/src/portal/DataImportView.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import Api, { DataImportItem } from 'api/api'
+import LoadingSpinner from 'util/LoadingSpinner'
+import { ColumnDef, getCoreRowModel, getSortedRowModel, SortingState, useReactTable } from '@tanstack/react-table'
+import { basicTableLayout, DownloadControl, renderEmptyMessage, RowVisibilityCount } from 'util/tableUtils'
+import { currentIsoDate, instantToDefaultString } from '@juniper/ui-core'
+import { useLoadingEffect } from '../api/api-utils'
+import { renderPageHeader } from 'util/pageUtils'
+import { StudyEnvContextT, useStudyEnvParamsFromPath } from '../study/StudyEnvironmentRouter'
+import { Link, useParams } from 'react-router-dom'
+
+
+/** show the dataImportItem list in table */
+export default function DataImportView({ studyEnvContext }:
+                                         { studyEnvContext: StudyEnvContextT }) {
+  const [dataImportItems, setDataImportItems] = useState<DataImportItem[]>([])
+  const [sorting, setSorting] = React.useState<SortingState>([{ 'id': 'createdAt', 'desc': true }])
+  const importedDate = instantToDefaultString(dataImportItems[0]?.createdAt)
+  const columns: ColumnDef<DataImportItem>[] = [
+    {
+      header: 'EnrolleeId',
+      accessorKey: 'createdEnrolleeId',
+      cell: ({ row }) => {
+        const enrolleIdLast8 = row.original?.createdEnrolleeId?.slice(-8)
+        if (row.original.status == 'DELETED') {
+          return <p>detail-{enrolleIdLast8}</p>
+        } else {
+          return <Link to={`${studyEnvContext.currentEnvPath}/participants/${row.original.createdEnrolleeId}`}
+            className="me-1">view detail-{enrolleIdLast8}</Link>
+        }
+      }
```

Comment:
If an entry has `FAILED` status, this is still clickable and an error is thrown in the UI because there isn't an enrolleeId. I think we should disable the link in that case